src/lib/remote_lib_executor.ts

```diff
@@ -0,0 +1,47 @@
+import {OpenCvLib} from './opencv';
+import {RemoteLib} from './remote_lib';
+import {decode} from '../util';
+import {pipe} from 'fp-ts/function';
+import * as D from 'io-ts/Decoder';
```

Comment:
just fyi, we have switched to the `zod` library to do decoding in algoreafrontend, a bit nicer and more lightweight

Reply:
Oh perfect, that looks much easier to use indeed! I'll to a separate PR to switch to zod